content/backstage/plugins/azure-devops.md

```diff
@@ -0,0 +1,234 @@
+---
+humanName: Azure Devops
+heading: 'Azure DevOps Plugin'
+lead: "A view of the latest Pull Requests and Builds for your components in Backstage"
+attribution:
```

Comment:
Let's remove this page please. It discusses setting up Azure DevOps on a self-hosted Backstage instance, which is not really in scope for the ticket these docs are part of.

It is beneficial to cover the self-hosted installation perspective, but if we do that I would prefer that we write original content rather than copy/pasting the [README of the plugin](https://github.com/backstage/backstage/tree/master/plugins/azure-devops).

Reply:
Ok sorry - I thought we wanted to have all our available plugins represented on this page. Is the original content bit for SEO?